conduit/pipeline.go

```diff
@@ -187,56 +200,81 @@ func (p *pipelineImpl) Init() error {
 
 	// TODO Need to change interfaces to accept config of map[string]interface{}
 
-	// Initialize Exporter first since the pipeline derives its round from the Exporter
-	exporterLogger := log.New()
-	exporterLogger.SetFormatter(makePluginLogFormatter(plugins.Exporter, (*p.exporter).Metadata().Name()))
-
-	jsonEncode := string(json.Encode(p.cfg.Exporter.Config))
-	err := (*p.exporter).Init(plugins.PluginConfig(jsonEncode), exporterLogger)
-	exporterName := (*p.exporter).Metadata().Name()
+	// load pipeline metadata
+	var err error
+	p.blockMetadata, err = p.loadBlockMetadata()
```

Comment:
nit: `initializeOrLoadBlockMetadata` might be a better name. While reading, I wondered how the metadata file would be initialized.

Reply:
Is it possible to initialize this after the importer, so that it can have access to the genesis file?